p/themes/BlueLagoon/icons/icon.svg

```diff
@@ -1,34 +1 @@
-<!-- Created with Inkscape (http://www.inkscape.org/) -->
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
The reason that resizing isn't working is because you don't have a viewBox set. Instead of replacing the entire SVG and setting the exact size, you can add a viewBox to this line and then it'll scale properly with CSS. The bit you need to add is ` viewBox="0 0 59.4 59.7"`, so your resulting line will look like: 
```
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 59.4 59.7">>>
```

Reply:
Thanks ! I just force-pushed on my branch sorry, I lost your commit :s

Reply:
Nevermind, I thought you added a commit to this MR but you only referenced it -_-.
Thanks again for the tip